plugins/huggingface/newhelm/suts/huggingface_client.py

```diff
@@ -517,4 +517,4 @@ def translate_response(
         return SUTResponse(completions=sut_completions)
 
 
-SUTS.register("gpt2", HuggingFaceSUT, "gpt2")
+SUTS.register("huggingface/gpt2", HuggingFaceSUT, "gpt2")
```

Comment:
I do think we need to coalesce on a story here, but I'm not sure what direction we should go.

1. The registries are really only there to make command line selection easier. Any library user of newhelm can make and use any SUT they want without registering them. That includes using whatever naming convention they want.
2. I *think* that for the purpose of command line running, we probably don't want multiple versions of the same SUT. While there is a benefit if someone would prefer to run a model remotely/locally, I think that is outweighed by the potential confusion of "which version should I be using?"
3. HELM resolved this conflict by introducing the concept of "deployment". If we think it is time to start having multiple ways to run the same model, I think we should discuss bringing that concept into newhelm.

Reply:
Yeah, I think this is one of the areas where us having an opinion is useful to the user, especially our type 2 user. We want them to easily be able to download this, run it with some basic args, and see interesting things happen. We should have an opinion on the best way to run common models using NewHELM and then make that the default.